src/main/assembly/dist/res/localrun.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+USER_HOST="..." # to be filled in by the developer; this is the 'user@host' for Vagrant
+PRIVATE_KEY="..." # to be filled in by the developer; this is the private key for ssh-ing into Vagrant
+
+ssh -oStrictHostKeyChecking=no -i ${PRIVATE_KEY} ${USER_HOST} weasyprint -e utf8 -f pdf - -
```

Comment:
I see your concern, but this is not expected, because you fill in these values in a *copy* of this folder called `home` which is commonly listed in the `.gitignore` file. Obviously this could go wrong, so if you got suggestions to improve this, it would be highly appreciated!